lib/token-util.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs')
+const get = require('simple-get')
+const os = require('os')
+const path = require('path')
+const rimraf = require('rimraf')
+
+const baseUrl = 'https://test-wprl.auth.us-east-2.amazoncognito.com/'
+const clientId = '3h21816o70ts9c28vu8guopa07'
+const dotFileName = path.join(os.homedir(), '.clinic')
+const tokenEndpoint = `${baseUrl}/oauth2/token`
+// const userInfoEndpoint = `${baseUrl}/oauth2/userInfo`
+
+module.exports = { clientId, loadTokens, refreshTokens, removeTokens, saveTokens }
+
+function loadTokens (cb) {
+  fs.readFile(dotFileName, function (err, text) {
+    if (err) return cb(err)
+
+    try {
+      cb(null, JSON.parse(text))
```

Comment:
don't try-catch the cb. might end up calling the cb twice if the cb itself throws